forum/qa-include/app/users.php

```diff
@@ -141,6 +141,8 @@ function qa_start_session()
 			@ini_set('session.gc_maxlifetime', 86400); // worth a try, but won't help in shared hosting environment
 			@ini_set('session.use_trans_sid', false); // sessions need cookies to work, since we redirect after login
 			@ini_set('session.cookie_domain', QA_COOKIE_DOMAIN);

```

Comment:
No niestety serio, jak napisałem wyżej nie poprawiałem nic tylko zduplikowałem linie i zmieniłem co trzeba. Do php.ini nie mam dostępu, to współdzielony. Poza tym chcę odczytywać to z tych stałych, aby jedna zmiana działała w całym kodzie.